extensions/wikia/Search/classes/Services/ESEntitySearchService.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Wikia\Search\Services;
+
+
+abstract class ESEntitySearchService extends AbstractSearchService {
```

Comment:
What is the purpose of this class? Other than being the super class of `ESFandomSearchService` what does it bring to the table?